### DISCUSSION TITLE
Error with JSON formatting when using Bitcoin Core RPC server

### DISCUSSION THREAD

**Discussion by Emgevorgyan** [ordinals/ord#1937] [Q&A]
When attempting to connect to the Bitcoin Core RPC server using the `ord` wallet and perform the wallet create operation, an error is encountered with the JSON formatting of the request. Specifically, the server is expecting a map but receiving a sequence instead, resulting in a "JSON error: invalid type" message.

I'm running Ubuntu 22.04 and using the following command:

command: 
`RUST_LOG=debug ./ord --cookie-file cookie-file-path wallet create`

output:
`
DEBUG bitcoincore_rpc] JSON-RPC request: getblockchaininfo []
DEBUG bitcoincore_rpc] JSON-RPC request: getnetworkinfo []
error: JSON error: invalid type: sequence, expected a map
`

Has anyone else encountered this issue or have any suggestions for how to fix it? I would appreciate any help or advice you can provide. Thanks!

**Comment by Grin0502** [comment]
same error
